packages/rspack-cli/src/utils/options.ts

```diff
@@ -45,11 +45,10 @@ export const commonOptions = (yargs: yargs.Argv) => {
 				string: true,
 				describe: "Name of the configuration to use."
 			},
-			'disable-interpret': {
-				type: "boolean",
-				default: false,
-				describe: "Disable interpret for loading the config file.",
-				alias: "d"
+			'config-loader': {
+				type: "string",
+				default: "native",
+				describe: "Specify the loader to load the config file, can be `none` or `native`.",
```

Comment:
My recommendation:

- `native` - use the native Node.js behaviour
- `register` - use any register installed in the user project

Defaults to `register` as before.

Reply:
By register you mean interpret?

Reply:
yep, `interpret` is the library name and `register` could be a more accurate term for these registers

Reply:
@chenjiahan how i use this CLI option for disable any loaders?

Reply:
`--config-loader native` to disable the loaders and use Node.js native behavior

Reply:
What dou you mean by native? strip types? I think this value doesn't make it clear what exactly will happen

Reply:
Hmm, I checked it again now and realized they are a bit confusing:

The option is named as "config-loader" and as I understand it means what loader will be used

And, "native" sounds like "native Node.js loader will be used"

But in Node.js there is no default loaders for modules, there is only API for register custom loaders

`--config-loader native` sound like there is a some native loader for typescript for example

We don't say that importing regular JS file is "using native Node.js loader"

Reply:
Unfortunately, it is already done in Vite and Rsbuild so I updated PR according this tools

Reply:
> But in Node.js there is no default loaders for modules

When we say "native loaders", we mean the ESM loaders or CJS loaders inside Node.js:

- https://github.com/nodejs/node/blob/3186468bed7c40b9a76a69643468d8e5a16c8771/lib/internal/modules/cjs/loader.js
- https://github.com/nodejs/node/blob/3186468bed7c40b9a76a69643468d8e5a16c8771/lib/internal/modules/esm/loader.js

I think the "native" here does not only refer to Node.js. Bun and Deno also provide native support for TypeScript.

Reply:
Just want to say that it sound strange: 
When I pass "register" value instead "native" to "--config-loader" option it looks like i decline "native" and disable native loader

But in fact it doesn't work that way: under the hood there is no way to disable native runtime feature to importing some modules, there is only ability to override some default behevior

Reply:
I completely understand your confusion. This option should be removed in the future, when all Node.js versions used by Rspack users support TypeScript. It will be much easier for users to understand then.